### ISSUE TITLE
Table Caption

### ISSUE THREAD

**Issue by timhagemann** [WindForS/CorporateLatex#4]
Tables caption numbers do not include chapter numbers. Hence, the document looks like this:
Table 1 ...
Table 2 ...
Table 1 ...

Is this a bug or a feature?

**Comment by AndyClifton** [comment]
> Is this a bug or a feature?

Good question. Let's go with bug.

I've checked the report template file and can see it in Section 4.4 where the figure number is 1 (even though this would be the second figure).

**Comment by AndyClifton** [comment]
Caption numbering is controlled by the [`chngcntr`](https://ctan.org/pkg/chngcntr?lang=en) package. At the moment, the style file includes...

```
% number figures and tables continuously across chapters
\RequirePackage{chngcntr}
\counterwithout{figure}{section}
\counterwithout{table}{section}
```

**Comment by AndyClifton** [comment]
N.B. The required behaviour is for figures to increment by one but include the chapter number.

**Comment by AndyClifton** [comment]
@timhagemann  thank you for the suggested revision in your pull request!

Actually, fixing this issue brought some other problems to my attention, which I also fixed. 

These fixes are all included in [release 1.3](https://github.com/WindForS/CorporateLatex/releases/tag/v1.3) of the `windfors.sty` package, which is directly backward compatible with the version you were using. I suggest updating to this release when you get the chance.
